src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -1365,6 +1365,19 @@ protected override void Dispose(bool disposing)
 
                 public override ValueTask<int> ReadAsync(Memory<byte> buffer, CancellationToken cancellationToken) => throw new NotSupportedException();
 
+                public override void Write(byte[] buffer, int offset, int count)
+                {
+                    ValidateBufferArgs(buffer, offset, count);
+                    Http2Stream? http2Stream = _http2Stream;
+                    if (http2Stream == null)
+                    {
+                        throw new ObjectDisposedException(nameof(Http2WriteStream));
+                    }
+
+                    // Sync-over-async.  See comment on Http2Connection.SendAsync.
+                    http2Stream.SendDataAsync(new ReadOnlyMemory<byte>(buffer, offset, count), default).AsTask().GetAwaiter().GetResult();
```

Comment:
nit: code review harder when non-obvious `default` is used.

```suggestion
                    http2Stream.SendDataAsync(new ReadOnlyMemory<byte>(buffer, offset, count), cancellationToken: default).AsTask().GetAwaiter().GetResult();
```